### PR TITLE
CCD-3807 ts integration 

### PR DESCRIPTION
**Pull request by nathan-clark**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by rajaram-nagarajan — DISMISSED** [review]
LGTM

**Review by danlysiak — APPROVED** [review]

File: k8s/namespaces/ts/ts-translation-service-int/demo.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ts-translation-service-int
+spec:
+  values:
+    java:
+      image: hmctspublic.azurecr.io/ts/translation-service:prod-00a4d27-20220530161141 #{"$imagepolicy": "flux-system:ts-translation-service"}
```
Comment (thomast1906) [inline k8s/namespaces/ts/ts-translation-service-int/demo.yaml:8]: is demo to use teh demo image policy you are adding ?

Comment (nathan-clark) [reply k8s/namespaces/ts/ts-translation-service-int/demo.yaml:8]: No I assume -int will be diff to OG demo

**Review by thomast1906 — APPROVED** [review]
Approving for now, did mention to Nathan some chats are happening to see if -int env is to be used. He will check with DM

**Pull request merged**